src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 ++++++
 * webapp backup/restore: Fix issue where restore command fails because of a null reference
 * appservice: support default app service plan through `az configure --defaults appserviceplan=my-asp`
+* appservice: list-location: reports Free as an invalid SKU
```

Comment:
This should be in the new version section, the current version will be released tomorrow.
Also the text should reflect what is being fixed, e.g. `fix the bug that reports Free as an invalid SKU `

Reply:
@yugangw-msft  & @tjprescott , i will make the change move to new version & update history. However, there is no existing test for list-locations, since this returns a list of locations from the API, & that is not a static list - so can we just check the response is 200, specifically for Free SKU? Let me know. Thanks!

Reply:
You can verify the response is the same as from `F1`